pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc

```diff
@@ -0,0 +1,660 @@
+= Configuring JWT Library Functions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+NOTE: To view an example policy project that uses PDK's JWT Library, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/main/jwt-validation/README.md[JWT Validation Policy Example^].
+
+JSON Web Token (JWT) is a URL-secure method of representing claims to be transferred between two parties. The JWT token contains claims encoded in a JSON object as either the payload of a JSON Web Signature (JWS) or as a JSON web encryption (JWE) structure in plain text. This enables the claims to be digitally signed and protected with a message authentication code (MAC). Because the token is signed, you can trust the information and its source.
+
+The Flex Gateway Policy Development Kit (PDK) JWT library provides a set of tools to validate the JWT signatures and to extract and validate the claims of all incoming requests.
```

Comment:
Maybe we can omit the "Flex Gateway" and simply use "Policy Development Kit".

Reply:
Ehhh, I actually don't think we can. My current understanding is that Flex Gateway Policy Development Kit is the official name and the PDK is the shorthand.

I agree with you, but, I think it has to stay as is.

Reply:
No problem, it was just a suggestion